modules/installing-ocp-agent.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent_{context}"]
+= Installing {product-title} with Agent
+WIP
+You can use the Agent-based installation to install a:
+
+* Single-node cluster
+* Compact cluster
+* Highly Available (HA) cluster
+
+The following procedure deploys a single-node {product-title} and you can use this procedure as a basis and modify according to your requirements.
+
+.Prerequisites
+* Golang version 1.17 and greater
```

Comment:
Is golang really a prerequisite to using the the installer?

Reply:
Not required , removing it as a pre-req